packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -818,7 +834,7 @@ export class DeltaManager<TConnectionManager extends IConnectionManager>
             throw new Error("Attempted to process an inbound message without a handler attached");
         }
         this.handler.process(message);
-
+        this.opsCurrentlyProcessing = false;
```

Comment:
Logic question here - can you explain why this is reset before emitting the "op" event?  Does that order matter?  Naively I'd expect this to be the absolute last thing in the function, but if there's a reason it should be reset before emitting that event then that would be good to explain in a comment.  Given the nature of this PR, I wouldn't be surprised if there were some subtlety here.

Reply:
In my understanding, the flag should be reset after the above `process` function is called to signify that op processing was complete.